Casks/r/roonbridge.rb

```diff
@@ -1,16 +1,16 @@
 cask "roonbridge" do
-  version "1.0"
-  sha256 :no_check
+  version "100801125"
+  sha256 "003dcaf79a3ae4bc06f62e5edd35bb40ccf32b05df28bf59182eacb9ac35ba28"
 
-  url "https://download.roonlabs.net/builds/RoonBridge.dmg",
-      verified: "download.roonlabs.net/builds/"
+  url "https://download.roonlabs.net/updates/stable/RoonBridge_#{version}.dmg",
+      verified: "download.roonlabs.net/updates/"
   name "Roon Bridge"
   desc "Music player network extender"
-  homepage "https://roonlabs.com/"
+  homepage "https://roon.app/"
```

Comment:
```suggestion
  homepage "https://roon.app/en/"
```

The current homepage redirects to `https://roon.app/en/`.

Reply:
I think it is based on the locale, my case it is going to /ko. Should I set to the default to /en?

Reply:
Thank you, let’s keep it as is then.